docs/make.jl

```diff
@@ -1,4 +1,5 @@
-using ImageSmooth
+push!(LOAD_PATH,"../src/")
```

Comment:
You should instead add this package to `docs/Project.toml` via developing mode.

For example, in your project folder: `julia --project=docs` and then `pkg> dev .` This will add `ImageSmooth` as a dependency to `docs/Project.toml` (and also to `docs/Manifest.toml`) using your local version of ImageSmooth.

This is exactly what the documentation CI is doing to setup the environment: https://github.com/johnnychen94/ImageSmooth.jl/blob/d78763689b6a9a4056461b12f01eb6f3af8464a4/.github/workflows/UnitTest.yml#L50-L54 

Also remember to 1) remove `docs/Manifest.toml` and 2) add `docs/Manifest.toml` to `.gitignore` so that you don't accidently commit this file in the future.